tests/jobs/test_backfill_job.py

```diff
@@ -160,10 +160,11 @@ def test_trigger_controller_dag(self):
         )
         job.run()
 
-        task_instances_list = dag_file_processor._process_task_instances(
-            target_dag,
-            dag_runs=DagRun.find(dag_id='example_trigger_target_dag')
-        )
+        task_instances_list = []
```

Comment:
Is it necessary?

Reply:
Er, no it doesn't look right. Will take a look.

Reply:
This one is left I think @ashb?

Reply:
Yes, working on this now.

Reply:
Fixed in ce51c264f

Reply:
I don't see change. Can you check it?